.eslintrc

```diff
@@ -7,32 +7,39 @@
   "plugins": ["@typescript-eslint", "eslint-plugin-import"],
   "extends": ["plugin:@typescript-eslint/recommended"],
   "rules": {
+    "@typescript-eslint/class-name-casing": "error",
+    "@typescript-eslint/explicit-function-return-type": ["error", {
+        "allowExpressions": true
+    }],
+    "@typescript-eslint/func-call-spacing": "error",
     "@typescript-eslint/indent": ["error", 2],
+    "@typescript-eslint/interface-name-prefix": ["error", "always"],
     "@typescript-eslint/no-require-imports": "error",
-    "@typescript-eslint/interface-name-prefix": "off",
-    "@typescript-eslint/no-use-before-define": "off",
     "@typescript-eslint/no-unused-vars": ["warn", {
         "varsIgnorePattern": "^_"
     }],
-    "@typescript-eslint/explicit-function-return-type": ["warn", {
-        "allowExpressions": true
+    "@typescript-eslint/no-use-before-define": "off",
+    "@typescript-eslint/semi": "error",
+    "@typescript-eslint/type-annotation-spacing": "error",
+    "import/no-extraneous-dependencies": ["error", {
+        "devDependencies": false,
+        "optionalDependencies": false,
+        "peerDependencies": false
+    }],
+    "camelcase": "error",
```

Comment:
I don't think its enforcing constants, but it does allow constants

Reply:
Hmm , does it work then? because I saw there is typescript versiomn of camelcase and in recommended settings they disable this rule and enable typescript version of it